### PR TITLE
Update CrossVersionUtil.scala

### PR DESCRIPTION
**Pull request by HelloKunal**
Change toInt in toLong ( #7361 )
https://github.com/sbt/sbt/issues/7361

### CODE REVIEW

**Review by eed3si9n — APPROVED** [review]
Thanks, the change looks good to me.

File: core/src/main/scala/sbt/internal/librarymanagement/cross/CrossVersionUtil.scala [hunk 1]
```diff
@@ -66,7 +66,7 @@ object CrossVersionUtil {
   private[sbt] def scalaApiVersion(v: String): Option[(Long, Long)] = v match {
     case ReleaseV(x, y, _, _)                     => Some((x.toLong, y.toLong))
     case BinCompatV(x, y, _, _, _)                => Some((x.toLong, y.toLong))
-    case NonReleaseV_1(x, y, z, _) if z.toInt > 0 => Some((x.toLong, y.toLong))
+    case NonReleaseV_1(x, y, z, _) if z.toLong > 0 => Some((x.toLong, y.toLong))
```
Comment (eed3si9n) [inline core/src/main/scala/sbt/internal/librarymanagement/cross/CrossVersionUtil.scala:69]: The build seems to be failing at Scalafmt

**Comment by eed3si9n** [comment]
@HelloKunal Could you sign the Scala CLA plz - https://www.lightbend.com/contribute/cla/scala

**Comment by HelloKunal** [comment]
@eed3si9n I have signed

**Pull request merged**